packages/test/test-end-to-end-tests/src/test/messageSize.spec.ts

```diff
@@ -66,16 +66,19 @@ describeNoCompat("Message size", (getTestObjectProvider) => {
 
 		// Create a Container for the first client.
 		localContainer = await provider.makeTestContainer(configWithFeatureGates);
-		dataObject1 = await requestFluidObject<ITestFluidObject>(localContainer, "default");
-		dataObject1map = await dataObject1.getSharedObject<SharedMap>(mapId);
+		localDataObject = await requestFluidObject<ITestFluidObject>(localContainer, "default");
+		localMap = await localDataObject.getSharedObject<SharedMap>(mapId);
 
 		// Load the Container that was created by the first client.
 		remoteContainer = await provider.loadTestContainer(configWithFeatureGates);
-		dataObject2 = await requestFluidObject<ITestFluidObject>(remoteContainer, "default");
-		dataObject2map = await dataObject2.getSharedObject<SharedMap>(mapId);
+		remoteDataObject = await requestFluidObject<ITestFluidObject>(remoteContainer, "default");
+		remoteMap = await remoteDataObject.getSharedObject<SharedMap>(mapId);
+
 		await waitForContainerConnection(localContainer, true);
 		await waitForContainerConnection(remoteContainer, true);
 
+		// Force the local container into write-mode by sending a small op
+		localMap.set("test", "test");
```

Comment:
It should already be in write mode when the test starts, right? When we wait for it to be synchronized, it will try to push the current batch (this small op), then disconnect and turn to write mode and send the batch.